tests/test_modeling_flax_common.py

```diff
@@ -67,6 +67,23 @@ def random_attention_mask(shape, rng=None):
     return attn_mask
 
 
+def floats_tensor(shape, scale=1.0, rng=None, name=None):
+    """Creates a random float32 tensor"""
+    if rng is None:
```

Comment:
This function already exists on master. You should rebase the branch with master.